tests/e2e/helpers.mjs

```
@@ -125,7 +132,15 @@ const runWithOpenWebviewIfAppRoot = async (webview, runInWebview) => {
     await runInWebview();
     return true;
   } finally {
-    await webview.close();
+    try {
+      await webview.close();
+    } catch (error) {
+      // VS Code may have already torn down the webview iframe during editor
+      // transitions; WebdriverIO then surfaces this as "invalid session id".
+      if (!(error instanceof Error) || !error.message.includes('invalid session id')) {
+        throw error;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)